packages/@angular/cli/commands/completion.ts

```diff
@@ -117,14 +117,15 @@ const CompletionCommand = Command.extend({
         });
       }
 
-      let opts = '';
+      let opts: string[] = [];
       if (command.blueprints && command.blueprints[0]) {
-        opts += extractBlueprints(command.blueprints);
+        opts = opts.concat(extractBlueprints(command.blueprints));
       }
 
       if (command.availableOptions && command.availableOptions[0]) {
```

Comment:
sure, I can remove the comment lines.

WIth the shell test cases - do you mean testing it by myself (which I do before checking in) or adding it somewhere to tests? I need some advice here...

btw, with both bash and zsh, the following syntax also works with the comments:
```
. <(ng completion)
```